src/snowflake/connector/result_batch.py

```diff
@@ -290,7 +289,6 @@ def _download(
                         "url": chunk_url,
                         "headers": self._chunk_headers,
                         "timeout": DOWNLOAD_TIMEOUT,
-                        "stream": True,
```

Comment:
By not having a stream we force the Exception to occur in this code block

Reply:
Does this have a performance impact?

Reply:
I don't think so, I think if anything was impacted I think it would be the memory usage.
I kicked off 2 perf tests just to verify this:

https://jenkins.int.snowflakecomputing.com/job/ClientPerfRunner/8200/ for AWS
https://jenkins.int.snowflakecomputing.com/job/ClientPerfRunner/8201/ for GCP